web/docs/setup/tune.md

```diff
@@ -230,45 +173,75 @@ vast:
-a zero exit code upon success, making it suitable for use in launch system
-scripts.
+VAST defaults to the `feather` store. To use the Parquet store, adjust
+`vast.store-backend`:
 
-:::note
-The server waits for ongoing import processes to terminate before shutting down
-itself. In case an import process is hanging, you can always terminate the
-hanging process manually to shutdown the server.
+```yaml
+vast:
+  plugins:
+    - parquet
+  store-backend: parquet
+```
+
+There's an inherent space-time tradeoff between the stores that affects CPU,
+memory, and storage characteristics. Compared to the Feather store, Parquet
+differs as follows:
+
+1. Parquet files occupy ~40% less space, which also reduces I/O pressure during
+   querying.
+2. Parquet utilizes more CPU cycles during ingest (~10%) and querying.
+
+Parquet has the major advantage that it's the de-facto standard for encoding
+columnar data in modern data architectures. This allows other applications that
+support reading from Parquet *native* access to the data.
+
+:::tip Recommendation
+
+Use Parquet when:
+
+1. Storage is scarce, and you want to increase data retention
+2. Workloads are I/O-bound and you have available CPU
+3. Reading data with with off-the-shelf data science tools is a use case
+
+For an in-depth comparison, take a look at our [blog post on Feather and
+Parquet](/blog/parquet-and-feather-writing-security-telemetry).
 :::
 
-## Logging
+VAST supports [rebuilding the entire database](#rebuild-partitions) in case you
+want to switch to a different store format. However, VAST works perfectly fine
+with a mixed-storage configuration, so a full rebuild is not required.
 
-The VAST server writes log files into a file named `server.log` in the database
-directory by default. Set the option `vast.log-file` to change the location of
-the log file.
+### Adjust the store compression
 
-VAST client processes do not write logs by default. Set the option
-`vast.client-log-file` to enable logging. Note that relative paths are
-interpreted relative to the current working directory of the client process.
+VAST compresses partitions using Zstd for partitions at rest. To fine-tune
+the space-time trade-off, VAST offers a setting, `vast.zstd-compression-level`
+to allow fine-tuning the compression level:
 
-Server log files rotate automatically after 10 MiB. The option
-`vast.disable-log-rotation` allows for disabling log rotation entirely, and the
-option `vast.log-rotation-threshold` sets the size limit when a log file should
-be rotated.
+```yaml
+vast:
+  zstd-compression-level: 1
+```
 
-VAST processes log messages in a dedicated thread, which by default buffers up
-to 1M messages for servers, and 100 for clients. The option
-`vast.log-queue-size` controls this setting.
+Currently, the default value is taken from Apache Arrow itself, using
+`arrow::util::Codec::DefaultCompressionLevel(arrow::Compression::ZSTD)`
```

Comment:
I think this is too much of an implementation detail; if you want to keep it, I would rather link to the documentation of the function than just giving its signature, which is rather meaningless.


Link to docs: https://arrow.apache.org/docs/cpp/api/utilities.html#_CPPv4N5arrow4util5Codec26UseDefaultCompressionLevelEv